backstop_data/engine_scripts/puppeteer/documentSelectionHelper.js

```diff
@@ -1,20 +1,11 @@
 module.exports = async (page, scenario) => {
-    if (scenario.hasOwnProperty('selectDocuments')) {
-        await page.goto(scenario.selectDocuments)
-
-        await page.waitFor('#select_documents_form_has_valid_passport')
-        await page.waitFor('#select_documents_form_has_driving_license')
-        await page.waitFor('#select_documents_form_has_phone_can_app')
-        await page.waitFor('#select_documents_form_has_credit_card')
-
-        await page.click('#select_documents_form_has_valid_passport')
-        await page.click('#select_documents_form_has_driving_license')
-        await page.click('#select_documents_form_has_phone_can_app')
-        await page.click('#select_documents_form_has_credit_card')
+    if (scenario.selectDocuments) {
```

Comment:
Maybe update the scenario to use the `aboutDocuments` key for consistency

Reply:
I'll remove this completely in the next PR when we don't need to click the button anymore